ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -471,6 +471,14 @@ FEDataManager::reinitElementMappings()
 {
     IBTK_TIMER_START(t_reinit_element_mappings);
 
+    // We reinitialize mappings after repartitioning, so clear the cache since
```

Comment:
This is fine, but I would suggest simply using `d_system_dof_map_cache->clear()`. As long as we always access via `::getSystemDofMapCache()` --- which we do currently --- then these will get set up automatically there as needed.